internal/cmd/build_test.go

```diff
@@ -57,6 +61,10 @@ func TestBuild(t *testing.T) {
 
 			r.NoError(os.Chdir(filepath.Join(dir, v.appname)))
 
+			out, err = testhelpers.RunBuffaloCMD(t, v.resourceargs)
```

Comment:
This imho should be its own test.

Reply:
I added this to make the test pass because without it, the `templates` directory doesn't have a subdirectory so my patch using `go:embed * */*` fails to build. I don't think we need another separated integration test which takes much more time, the purpose of the test still remains the same.